benchmarks/Benchmarks/CertificatesBenchmarks/Benchmarks.swift

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCertificates open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftCertificates project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftCertificates project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Benchmark
+import Sources
+
+let benchmarks = {
+    Benchmark("Verifier", configuration: .init(warmupIterations: 1)) { benchmark in
+        await verifier()
+    }
+    
+    let runParseWebPKIRoots = parseWebPKIRoots()
+    Benchmark("Parse WebPKI Roots") { benchmark in
+        runParseWebPKIRoots()
+    }
+    
+    Benchmark("TinyArray non-allocating functions") { benchmark in
```

Comment:
For the kind of very small (short run time) tests, I would suggest you use the support for `scaledIterations` with an inner loop to decrease the benchmark measurement overhead and get higher quality measurements, e.g.:

```swift
    Benchmark.defaultConfiguration.scalingFactor = .kilo
...
    Benchmark("TinyArray non-allocating functions") { benchmark in
        for _ in benchmark.scaledIterations {
            tinyArrayNonAllocationFunctions()
        }
    }
```
You can get the time measurements (and malloc stats etc) scaled to per-iteration using i.e.
```
swift package benchmark --scale
```